### PR TITLE
data: tarunclub

### PR DESCRIPTION
**Pull request by tarunclub**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5663"></a>

### CODE REVIEW

File: data/tarunclub.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+	"name": "Tarun Kumar",
+	"displayStatsPublic": true,
```
Comment (vedprakashnautiyal) [inline data/tarunclub.json:3]:
```suggestion
```

Comment (vedprakashnautiyal) [reply data/tarunclub.json:3]: This is not required .

**Review by vedprakashnautiyal — CHANGES_REQUESTED** [review]

**Review by khushalarora — APPROVED** [review]
Thank you for contributing  :tada:

**Review by mahfuztbt — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**